include/micm/configure/solver_config.hpp

```diff
@@ -116,6 +117,13 @@ namespace micm
           parameters_(parameters)
     {
     }
+    SolverParameters(System&& system, std::vector<Process>&& processes, RosenbrockSolverParameters&& parameters, double relative_tolerance)
+        : system_(system),
+          processes_(processes),
+          parameters_(parameters), 
+          relative_tolerance_(relative_tolerance)
```

Comment:
@montythind @K20shores since we move relative and absolute tolerances into the `State` class now, why we still need the `relative_tolerance_` here for the solver parameters?